### PR TITLE
feat: Added margin repay endpoint

### PR DESCRIPTION
**Pull request by gelito**
Added a new endpoint for repaying at margin trading.

Added also the interface.

I didn't attach test because you only be able to test it if you have assets with borrow or interests to repay.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -287,8 +287,12 @@ declare module 'binance-api-node' {
       useServerTime?: boolean
     }): Promise<CancelOrderResult>
     marginOpenOrders(options: { symbol?: string; useServerTime?: boolean }): Promise<QueryOrderResult[]>
+    marginRepay(options: { asset: string; amount:number; useServerTime?: boolean }): Promise<{tranId:number}>
   }
 
+
```
Comment (balthazar) [inline index.d.ts:293]: Can you remove that

**Review by balthazar — CHANGES_REQUESTED** [review]
Thanks for the PR! Can you also add the doc in the readme?

**Comment by gelito** [comment]
Added the doc in the readme.  I created a new section, `margin`, but I didn't include all previous methods.

**Comment by balthazar** [comment]
Thanks!

**Pull request merged**